checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -41,22 +57,23 @@
     extends GenericAnnotatedTypeFactory<CFValue, LockStore, LockTransfer, LockAnalysis> {
 
     /** Annotation constants */
-    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE;
+    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE, GUARDEDBYINACCESSIBLE, GUARDEDBY, GUARDEDBYBOTTOM, JCIPGUARDEDBY, JAVAXGUARDEDBY, GUARDSATISFIED;
```

Comment:
This long line is hard to read.
Why do you need fields for the javax and jcip versions of the annotations? Don't you just make them aliases and then never care about them?